Intel.MISP.Client.Process.yaml

```diff
@@ -47,18 +59,23 @@ sources:
     - name: GetIntel
       query: |
             LET CSVURL <= SELECT Tool_MISPTest_URL AS CSVURL FROM scope()
-            LET DLDir <= SELECT dirname(path=config.tempdir_windows) AS TD FROM scope() 
-            SELECT * FROM foreach(
-                row={
-                        SELECT * FROM clock(period=IntelCheckPeriod)}, 
-                query= {
-                        SELECT copy(accessor="data", filename=Content, dest=DLDir.TD[0] + "/" + Tool_MISPTest_FILENAME) AS FullPath 
-                        FROM http_client(url=CSVURL[0].CSVURL)})
+            LET Data = SELECT Content FROM http_client(url=CSVURL[0].CSVURL,
```

Comment:
We need to keep the `copy()` here, .  It looks like you are trying to use `upload_directory()` here instead of grabbing the intel from the server. We'll still want to implement what we've implemented in the server artifact to get around the size limitations with something like:

```
SELECT copy(accessor="file", filename=Content, dest=http://DLDir.TD[0] + "/" + Tool_MISPTest_FILENAME) AS FullPath 
             FROM http_client(url=CSVURL[0].CSVURL,tempfile_extension=".csv")})
```